client/me/purchases/paths.js

```diff
@@ -46,6 +46,7 @@ export function confirmCancelDomain( siteName, purchaseId ) {
 	return managePurchase( siteName, purchaseId ) + '/confirm-cancel-domain';
 }
 
+// legacy path
```

Comment:
Are these still necessary? Can we replace their remaining uses with the new paths?

Reply:
I need to make sure I understand the suggestion. If we remove these completely, any old links to (for instance) `/payment/add` will break, right?